gems/aws-sdk-core/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Issue - Fix QuerystringBuilder to support querystring lists of different types per rest-json protocol.
```

Comment:
Nit - changelog entries should be meaningful to customers. QuerystringBuilder is a private API, I would word this differently, like "Add query string support for lists of booleans, floats, integers, and timestamps in the rest json protocol" or something like that.

Reply:
I always have trouble figuring out how to "word" changelog entries and this is a good tip!!!